src/sentry/search/eap/columns.py

```diff
@@ -147,6 +145,54 @@ def proto_type(self) -> AttributeKey.Type.ValueType:
         return constants.DOUBLE
 
 
+@dataclass
+class FunctionDefinition:
```

Comment:
Would it be possible to keep `FunctionDefinition` untouched and just introduce `FormulaDefinition`? since Functions will never have a `formula_resolver`?